samples/server/petstore/python-fastapi/src/openapi_server/models/order.py

```diff
@@ -98,6 +98,11 @@ def from_dict(cls, obj: Dict) -> Self:
         if not isinstance(obj, dict):
             return cls.model_validate(obj)
 
+        # raise errors for additional fields in the input
+        for _key in obj.keys():
+            if _key not in cls.__properties:
+                raise ValueError("Error due to additional fields (not defined in Order) in the input: " + _key)
+
```

Comment:
This is the sample code corresponding to the config I mentioned in https://github.com/OpenAPITools/openapi-generator/pull/21170#discussion_r2070817272.

This code is being updated to throw errors when there are additional properties, but since the old default value was `disallowAdditionalPropertiesIfNotPresent=true`, these models should have _already_ been throwing errors before I made my change.  I'm not sure what I'm missing here.  Is there some hidden third case for some languages when `disallowAdditionalPropertiesIfNotPresent` isn't explicitly set to a true or false?

Reply:
See https://github.com/OpenAPITools/openapi-generator/pull/21170#discussion_r2150690187